src/test/java/com/egov/core-services/requestPayload/user/update/updateUser.json

```diff
@@ -0,0 +1,53 @@
+{
+    "RequestInfo": {
+        "authToken": "#(authToken)"
+    },
+    "user": {
+        "id": 4657,
+        "userName": "EMP-1013-000335",
```

Comment:
do not hard code any values and read it from yaml files where you usually keep test data

Reply:
take care of across all files related to hard code fix